cl/corpus_importer/tasks.py

```diff
@@ -390,12 +391,14 @@ def get_and_save_free_document_report(
             nature_of_suit=row.nature_of_suit,
             cause=row.cause,
         )
+        # Wait for 0.5 seconds before creating the next row.
+        time.sleep(0.5)
```

Comment:
I doubt this is really the problem, since each report probably has only a couple dozen rows, but the other solution that comes to mind is to do them as bulk_create. Want to try that and we can see if that helps?

I also kind of don't think this should be slow, since it's a narrow small table that's regularly truncated, right?

Reply:
Yeah, changed it to add these rows in bulk!